besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1547,6 +1547,20 @@ public void maxpeersOptionMustBeUsed() {
     assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
   }
 
+  @Test
+  public void rpcMaxLogsRangeOptionMustBeUsed() {
+    final long rpcMaxLogsRange = 150L;
+    parseCommand("--rpc-max-logs-range", Long.toString(rpcMaxLogsRange));
+
+    verify(mockRunnerBuilder).rpcMaxLogsRange(longArgumentCaptor.capture());
+    verify(mockRunnerBuilder).build();
+
+    assertThat(longArgumentCaptor.getValue()).isEqualTo(rpcMaxLogsRange);
+
+    assertThat(commandOutput.toString(UTF_8)).isEmpty();
+    assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
+  }
+
```

Comment:
add a test for what happens if you don't specify this option (what's the default behaviour)

Reply:
I've removed the empty default and added a numeric one.